docs/3-tutorials/01-run-collator.md

```diff
@@ -1,5 +1,168 @@
+ 
+### How to Cancel a Scheduled Leave Request 
+It is possible to cancel a scheduled leave request until it is not actually executed. 
+
+The steps for cancelling a scheduled leave request are:
+- Retrieve the size of the candidate pool, as described in [How to Get the Size of the Candidate Pool](#how-to-get-the-size-of-the-candidate-pool) section.
+- On Polkadot.js Apps, select _Developer>Extrinsics_.
+- In _using the selected account_ field, select the collator account.
+- Select _parachainStaking_ and then _cancelLeaveCandidates_.
+- Set the _candidateCount_ field to the size of the candidate pool.
+- Submit the transaction. Follow the wizard and sign the transaction using the password you set for the account.
+
+## How to Change the Bond Amount
+A collator can increase or decrease the amount of its bond. While increasing the bond is a straightforward operation that has an immediate effect, decreasing the bond is a 2-step operation: first it is scheduled a bond-less operation, than, after a fixed delay called _candidateBondLessDelay_, the execution of the bond-less operation can be invoked. The bond-less request can also be canceled.
+
+A bond-less operation is only allowed if the remaining bond amount after the decrease is still greater or equal the minimum stake required.
+
+### How to Increase the Bond Amount
+
+The steps for increasing the bond amount are:
+- On Polkadot.js Apps, select _Developer>Extrinsics_.
+- In _using the selected account_ field, select the collator account.
+- Select _parachainStaking_ and then _candidateBondMore_.
+- Set the _more_ field to the additional bond value.
+- Submit the transaction. Follow the wizard and sign the transaction using the password you set for the account.
+
+The collator balance will be immediately changed but the increased bond will have effect on collator selection only starting from the next round. 
+
+### How to Schedule a Bond Less Request
+
+The steps for scheduling a bond-less request are:
+- On Polkadot.js Apps, select _Developer>Extrinsics_.
+- In _using the selected account_ field, select the collator account.
+- Select _parachainStaking_ and then _scheduleCandidateBondLess_.
+- Set the _less_ field to the amount to be unbonded.
+- Submit the transaction. Follow the wizard and sign the transaction using the password you set for the account.
+
+### How to Execute a Scheduled Bond Less Request
+
+The steps for executing a bond-less request are:
+- After having scheduled a bond-less request, wait until _candidateBondLessDelay_ rounds have passed. The value for _candidateBondLessDelay_ can be found using Polkadot.js Apps, under _Developer>Chain State>Constants_ tab, selecting _parachainStaking_, _candidateBondLessDelay_ and pressing the "+" button.
+- In _using the selected account_ field, select the account that will request the extrinsic. It is not needed to be the collator account, it can be any account.
```

Comment:
nitpick, but this is the only section where next steps are missing:
```
- On Polkadot.js Apps, select Developer>Extrinsics.
- In using the selected account field, select the collator account.
```

Reply:
Good catch, thanks!